### ISSUE TITLE
Eliminating the jQuery dependency

### ISSUE THREAD

**Issue by miguel-perez** [miguel-perez/smoothState.js#174]
I've been thinking a lot about performance, and how smoothState will fit into the future of Javascript. This is going to be a big step, but I wanted to open an issue to start a conversation going about this.

**Comment by erbridge** [comment]
I'd like to see this happen. The site I'm using smoothState.js on only really includes jQuery because of it.

**Comment by koole** [comment]
Yes please! The only reason I now load in jQuery is for smoothState.

**Comment by sunnysingh** [comment]
Any updates on this? I just found out about smoothState and don't want to include jQuery.

**Comment by patrickcate** [comment]
I would love to see this as well.

**Comment by miguel-perez** [comment]
Biggest thing that needs to get done before a rewrite is finishing the unit tests. That way we know nothing will break.

**Comment by raininja** [comment]
any progress on this? or are there alternatives already in place?
